src/compile/layout/header.ts

```diff
@@ -84,6 +84,28 @@ export function getTitleGroup(model: Model, channel: HeaderChannel) {
   };
 }
 
+export function labelAlign(angle: number) {
+  angle = ((angle % 360) + 360) % 360;
+  if (270 < angle || angle <= 45) {
+    return {align: {value: 'right'}};
+  } else if (135 <= angle && angle < 225) {
+    return {align: {value: 'left'}};
+  }
+  return {};
+}
+
+export function labelBaseline(angle: number) {
+  angle = ((angle % 360) + 360) % 360;
+  if (angle <= 22.5 || 225 <= angle && angle < 270) {
```

Comment:
parentheses ...

Reply:
The rule for this aalso doesn't seem very symmetric.  

Some quadrant is as detailed as multiple of 22.5, some only for 45 -- is this intended?

Reply:
Yes, I tested the hextants and chose whatever values looked best in general